Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG  BASE=ubuntu:19.10
```

Comment:
The "official" Dockerfile is in contrib/docker and can be built using `make docker-image`

Reply:
you're right, my aim was to launch the build process inside a temporary container to avoid being dependent from libs on the host where it is run (cf. "build" level of the PR's Dockerfile) and also avoid copying sources in a dir below (as https://github.com/skydive-project/skydive/blob/3bd6c51764d2fba40b9b365a10d562ee5719e978/.mk/dist.mk#L51 for the binary). As I'm not fluent with Skydive build process, may be I haven't done it the best way.

Reply:
No problem. We have such a Dockerfile in the repository of our CI scripts : https://github.com/skydive-project/skydive-ci/blob/master/scripts/Dockerfile.compile 
We have now fixed most of the CI issues. Could you please rebase your PR on top of master, without the Dockerfile ? Thanks a lot !